example/MyFirstDrawing/MyFirstDrawingUITests/BrushToolView/GroupEraseSizeViewUITests.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  GroupEraseSizeViewUITests.swift
+//  MyFirstDrawingUITests
+//
+//  Created by Stant Macmini n04 on 09/07/19.
+//  Copyright © 2019 Stant 02. All rights reserved.
+//
+
+import XCTest
+import StantDrawing
+@testable import MyFirstDrawing
+
+class GroupEraseSizeViewUITests: XCTestCase {
+    let app = XCUIApplication()
+    
+    override func setUp() {
+        continueAfterFailure = false
+        
+        app.launch()
+        
+        let btnErase = app.buttons["eraserfullIdentifier"]
+        btnErase.tap()
+    }
+    
+    override func tearDown() {}
```

Comment:
If you are not doing anything here, I think that you could delete it.

Reply:
Done